solidity/random-beacon/contracts/libraries/DKG.sol

```diff
@@ -71,6 +71,8 @@ library DKG {
         // Group creation is not in progress. It is a state set after group creation
         // completion either by timeout or by a result approval.
         IDLE,
+        // Group creation is warmed up and sortition pool is locked.
+        WARMED_UP,
```

Comment:
Maybe `AWAITING_SEED`? This is what is actually happening: DKG locked the pool and is awaiting seed from the random beacon when it's in this state.

Reply:
See https://github.com/keep-network/keep-core/pull/2691/commits/a29a5ad1f24125d53b37cef20129d6c5977764be